src/actionTypes/index.js

```diff
@@ -3,5 +3,5 @@ export const SET_CURRENT_USER = 'SET_CURRENT_USER';
 export const ADD_FLASH_MESSAGE = 'ADD_FLASH_MESSAGE';
 export const DELETE_FLASH_MESSAGE = 'DELETE_FLASH_MESSAGE';
 export const CLEAR_FLASH_MESSAGES = 'CLEAR_FLASH_MESSAGES';
-export const USER_SIGNUP = 'USER_SIGNUP';
-export const USER_SIGNUP_SUCCESS = 'USER_SIGNUP_SUCCESS';
+export const BOOKMARK_ARTICLE = 'BOOKMARK_ARTICLE';
+export const SIGN_UP_SUCCESS = 'SIGN_UP_SUCCESS';
```

Comment:
```suggestion
export const SET_CURRENT_USER = 'SET_CURRENT_USER';
```
USER_SIGNUP_SUCCESS is not in the auth reducer or any other reducer for that matter and this could cause bugs in the future.